src/client/driver/driver.js

```diff
@@ -248,17 +254,52 @@ export default class Driver extends serviceUtils.EventEmitter {
         const childWindowDriverLink = new ChildWindowDriverLink(e.window, e.windowId);
 
         this.childWindowDriverLinks.push(childWindowDriverLink);
+        this._ensureChildWindowClosedWatcher();
+    }
+
+    _ensureChildWindowClosedWatcher () {
+        if (this.checkChildWindowClosedIntervalId)
+            return;
+
+        this.checkChildWindowClosedIntervalId = nativeMethods.setInterval.call(window, () => {
+            for (const childWindowDriverLink of this.childWindowDriverLinks) {
```

Comment:
will it work in IE?

Reply:
We have a special babel transpiler for the `for-of` loop: https://github.com/DevExpress/testcafe/blob/master/package.json#L61

The following code:
```js
for (const childWindowDriverLink of this.childWindowDriverLinks) {
```

transpiled to
```js
for (var _i2 = 0, _childWindowDriverLin2 = _this2.childWindowDriverLinks, _length2 = _childWindowDriverLin2.length; _i2 < _length2; _i2++) {
var childWindowDriverLink = _childWindowDriverLin2[_i2];
```
```